site/docs/v1/tech/release-notes.adoc

```diff
@@ -49,21 +49,68 @@ include::docs/v1/tech/__database-migration-warning.adoc[]
 
 === New
 * JWT Vending machine
```

Comment:
Not sure how that could be capitalized.  JWT vending machine?  Could ask Dan/DevDocsReadme for clarity.  Goes for a few of the other bullets.  Get clarity on the casing.